tests/cephadm/test_cephadm_upgrade.py

```diff
@@ -44,6 +44,17 @@ def run(ceph_cluster, **kwargs) -> int:
     verify_image = bool(config.get("verify_cluster_health", False))
     orch = Orch(cluster=ceph_cluster, **config)
 
+    # Clean up the ceph-qe-scripts to prepare for the execution of RGW singlesite and multisite test suites.
+    cleanup_dir = "/home/cephuser/"
+    cleanup_dir_cmd = (
+        f"find {cleanup_dir} -type d \\( "
+        f"-name rgw-tests -o -name rgw-ms-tests -o -name venv \\) "
+        f"-exec rm -r {{}} +"
+    )
+
+    for node in ceph_cluster.get_nodes():
```

Comment:
After upgrade, how are we downloading ceph-qe-scripts repo to the nodes?

Reply:
Hi @tintumathew10 ,
our test sanity_rgw.py or sanity_rgw_multisite.py will clone the ceph-qe-scripts repo if dir is not present:
https://github.com/red-hat-storage/cephci/blob/main/tests/rgw/sanity_rgw_multisite.py#L395

Reply:
Thanks @hmaheswa